roles/install_from_source/meta/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+galaxy_info:
+  author: EDB
+  description: Download and installation of software from source
+  company: "EnterpriseDB"
+
+  # If the issue tracker for your role is not on github, uncomment the
+  # next line and provide a value
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Choose a valid license ID from https://spdx.org - some suggested licenses:
+  # - BSD-3-Clause (default)
+  # - MIT
+  # - GPL-2.0-or-later
+  # - GPL-3.0-only
+  # - Apache-2.0
+  # - CC-BY-4.0
+  license: BSD
+
+  min_ansible_version: "2.11"
+
+  # If this a Container Enabled role, provide the minimum Ansible Container version.
+  # min_ansible_container_version:
+
+  #
+  # Provide a list of supported platforms, and for each platform a list of versions.
+  # If you don't wish to enumerate all versions for a particular platform, use 'all'.
+  # To view available platforms and versions (or releases), visit:
+  # https://galaxy.ansible.com/api/v1/platforms/
+  #
+  platforms:
```

Comment:
I think we can set this for all, then fix up as we find problems.  I don't believe anything is distro specific at this time.

Reply:
I'll update that when I update all repo documentation, this setting doesn't impact anything